packages/meson.rb

```diff
@@ -1,32 +1,24 @@
 require 'package'
 
 class Meson < Package
-  description 'The Meson Build System'
-  homepage 'http://mesonbuild.com/'
+  description 'Meson is an open source build system meant to be both extremely fast and user friendly.'
+  homepage 'https://mesonbuild.com/'
```

Comment:
Thank you for updating the description.  It's nice to have a short description like this to adequately explain the package.